pkg/platform/model/checkpoints.go

```diff
@@ -32,16 +32,16 @@ type Language struct {
 }
 
 // GetRequirement searches a commit for a requirement by name.
-func GetRequirement(commitID strfmt.UUID, namespace Namespace, requirement string) (*gqlModel.Requirement, error) {
+func GetRequirement(commitID strfmt.UUID, nsType Namespace, requirement string) (*gqlModel.Requirement, error) {
```

Comment:
a Namespace and a namespace type are two distinct things. I fear this is sufficiently confusing to create bugs.